docs/content/dagster-cloud/deployment/serverless.mdx

```diff
@@ -76,10 +76,19 @@ dagster-cloud configure
 
 You can also configure the `dagster-cloud` tool noninteractively; see [the CLI docs](/dagster-cloud/developing-testing/dagster-cloud-cli#environment-variables-and-cli-options) for more information.
 
+Add `dagster-cloud` as a dependency to `my-dagster-project/setup.py`:
+
+```python
+  install_requires=[
+      "dagster",
+      "dagster-cloud",    # add this line
+      "dagster-airbyte",
```

Comment:
I'm not sure what happened with the GH suggestion I made, but this is still indented a bit too much - could you please fix this before merging? :)

Reply:
my mistake probably, fixed all the way to the left now.